sdk/RegionsRoute.ts

```diff
@@ -26,7 +26,7 @@ export namespace Regions {
        * Vendor id
        * Identifier of the cloud provider vendor.
        */
-      vendor?: "aws" | "gcp" | "hcloud";
```

Comment:
@Palabola - is there a way to enforce the SDK generator to use a consistent ordering?
most of the current diff is due to reordering strings (where the order doesn't matter at all)